src/OSPSuite.Core/Domain/Module.cs

```diff
@@ -36,15 +36,14 @@ public enum MergeBehavior
    public class Module : ObjectBase, IEnumerable<IBuildingBlock>
    {
       private readonly List<IBuildingBlock> _buildingBlocks = new List<IBuildingBlock>();
-
       private T buildingBlockByType<T>() where T : IBuildingBlock => _buildingBlocks.OfType<T>().SingleOrDefault();
       private IReadOnlyList<T> buildingBlocksByType<T>() where T : IBuildingBlock => _buildingBlocks.OfType<T>().ToList();
 
       /// <summary>
       ///    Module is a PKSim module if created in PKSim and the module version
       ///    matches the version when it was first imported
       /// </summary>
-      public bool IsPKSimModule => !string.IsNullOrEmpty(PKSimVersion) && Equals(Version, ModuleImportVersion);
+      public bool IsPKSimModule { get; set; }
```

Comment:
Hum.  Why is this change required? We may end up with version not matching and the flag set to true

Reply:
yeah well, from this story we are going to decouple the version from the IsPkSimModule, meaning that the version will be only for showing the simulation on Sync or not, but the PkSimModule will be just reflected by this property.
The reason for this, is that now, there are some propeties (the one mentioned on the original story) that should not change the Type of module (AKA IsPkSimModule) but should change the version.

Reply:
Well then update the property comment because it’s wrong

Reply:
Comment is misleading.  If the module was created in PKSim and something was changed then the flag is false but it was stjlll created in PK-Sim.   I don’t know what the first issue is about so I am not sure what constitute a PK-sim module 
@PavelBal can maybe explain

Reply:
well would you say this comment is ok?
"Module is a PKSim module at the time of creation in PKSim, can be updated by changing some properties on Mobi" ?

Reply:
Yeah, the issue is that some commands in MoBi will change the building block but should not convert the module to extension.

We are going to need the commands themselves to know if they should unset this flag.